types/types.hh

```diff
@@ -1002,3 +1004,14 @@ struct appending_hash<data_type> {
         feed_hash(h, v->name());
     }
 };
+
+class data_values_list : public std::vector<data_value> {
```

Comment:
data_value_list is more grammatically accurate.

Reply:
okay.  I see you used `data_values_vector` below in your example.
Which do you prefer?  (Both look good to me)

Reply:
I meant to write data_values_list to be consistent with what you wrote. 

Either data_value_vector or data_value_list are fine.